packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -150,8 +150,11 @@ public boolean isContentChanged() {
         void onOtherMediaButtonClicked(String mediaSource, boolean allowMultipleSelection);
     }
 
-    public interface OnMediaFilesEditorLoadRequestListener {
+    public interface OnMediaFilesCollectionBasedBlockEditorListener {
```

Comment:
Hi @mzorz for the name here, could we call it something like `OnMediaFilesCollectionListener` for consistency and put a comment above about the block editor? I am not so sure about the `BasedBlockEditor` part. This is a nitpick though, nonetheless sharing my thoughts so we can probably improve the naming here if that's possible.

Reply:
As agreed on chat, we need to convey this listener potentially serves other blocks based off a `mediaFiles` collection so, we'd choose to stay with this naming for now 👍

Reply:
Understood :)